news/13389.bugfix.rst

```diff
@@ -0,0 +1,2 @@
+Python scripts added as :file:`data` files (e.g. via ``setuptools``\ ’ ``scripts=`` parameter)
+now get their shebang modified like regular scripts, and no longer break for venv paths with spaces.
```

Comment:
I found this news item a little confusing compared to what you mention in https://github.com/pypa/pip/issues/13389

What it be correct to phrase it more like:

```
Python scripts added as \:file:`data` files (e.g. via `setuptools`\ ’ `scripts=` parameter)
now have their shebangs modified *in the same way as regular scripts*, fixing issues with
venv paths that contain spaces.
```

?

Reply:
I would prefer to stick more closely to the terminology established in the wheel spec (which simply calls these "scripts") and say something straightforward like

> Rewriting of `#! python` headers in scripts has been improved to handle unusual paths more robustly.

I don't think we should say "like regular scripts" unless we are willing to guarantee that we use the same rewriting code (and currently, that would mean using an undocumented internal distlib API, which as I've said elsewhere I'm uncomfortable with - and even if we *have* to do so, I don't want to commit to that in the docs).